Cargo.lock

```diff
@@ -1044,8 +1045,8 @@ dependencies = [
 
 [[package]]
 name = "matrix-sdk-common"
-version = "0.8.0"
-source = "git+https://github.com/matrix-org/matrix-rust-sdk?branch=release-for-crypto-wasm-12#37c17cf854a70fe6f719c7fde49a6b0e4402988f"
+version = "0.9.0"
+source = "git+https://github.com/matrix-org/matrix-rust-sdk#b18100228e0456973d56d4ac6a58caf30b17bd8d"
```

Comment:
This will have a conflict with https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/184 but you can just keep your changes and ignore that.